pms/lib/snt/pms/rpc/user.rb

```diff
@@ -0,0 +1,14 @@
+module SNT
+  module PMS
+    module RPC
+      class User < Base
+        class << self
+          # ::SNT::PMS::RPC::User.authenticate_api_user(1)
```

Comment:
Only thing I would say is that this is suppose to act as an SDK that anyone can use and we should probably have some information about the params, since it isn't intuitive. At minimum, I would recommend we indicate what param keys are available.